pkg/infra/iac2/templates_compiler.go

```diff
@@ -407,6 +407,10 @@ func (tc TemplatesCompiler) handleIaCValue(v core.IaCValue) (string, error) {
 		return fmt.Sprintf("%s.arn", tc.getVarName(v.Resource)), nil
 	case string(core.ALL_BUCKET_DIRECTORY_IAC_VALUE):
 		return fmt.Sprintf("pulumi.interpolate`${%s.arn}/*`", tc.getVarName(v.Resource)), nil
+	case string(core.HOST):
+		return fmt.Sprintf("pulumi.interpolate`%s.cacheNodes[0].address`", tc.getVarName(v.Resource)), nil
+	case string(core.PORT):
+		return fmt.Sprintf("pulumi.interpolate`%s.cacheNodes[0].port`", tc.getVarName(v.Resource)), nil
```

Comment:
This needs to make sure that the resource is redis specific, because grabbing the host and port are going to be different pulumi code. Im thinking using the sub templating weve built may be better than these iac values now. Not a blocker for this pr but i think were going to want to rethink this